src/view/screens/Search/Search.tsx

```diff
@@ -1006,7 +1007,7 @@ let AutocompleteResults = ({
             }
             style={{borderBottomWidth: 1}}
           />
-          {autocompleteData?.map(item => (
+          {autocompleteData?.sort(sortProfiles).map(item => (
```

Comment:
same, can't mutate while rendering. doing it in the query before returning is ok.